packages/dashboard/src/DashboardLayout.tsx

```diff
@@ -343,14 +344,16 @@ export function DashboardLayout({
       {isDashboardEmpty && emptyDashboard}
       {layoutChildren}
       {React.Children.map(children, child =>
-        child != null
-          ? React.cloneElement(child as ReactElement, {
+        child != null ? (
+          <ErrorBoundary fallback={[]}>
```

Comment:
Add a comment that the fallback is so we do not show the error message over the entire app and we should do something like a toast message w/ the error in the future